rpython/rlib/rvmprof/cintf.py

```diff
@@ -20,12 +20,14 @@ class VMProfPlatformUnsupported(Exception):
 
 # vmprof works only on x86 for now
 IS_SUPPORTED = False
+NATIVE_PROFILING_SUPPORTED = False
 if sys.platform in ('darwin', 'linux', 'linux2') or sys.platform.startswith('freebsd'):
     try:
         proc = detect_cpu.autodetect()
         IS_SUPPORTED = (proc.startswith('x86')
                         or proc == 'aarch64'
                         or proc == 'riscv64')
+        NATIVE_PROFILING_SUPPORTED = proc.startswith('x86')# or proc.startswith('ppc-64')
```

Comment:
what about the commented out part? I suspect it should just be removed?

Reply:
Native profiling seems to be supported on powerpc https://github.com/pypy/pypy/blob/f67cb822600ddfc00e4fa72b3ef8251c386087cb/rpython/rlib/rvmprof/src/shared/vmprof.h#L51-L54

But it also seems like vmprof is not supported on powerpc in general.
So yes, it can be removed